core/src/toga/window.py

```diff
@@ -123,6 +124,31 @@ def __init__(
 
         self.on_close = on_close
 
+    @property
+    def screen(self) -> Screen:
+        """Instance of the :class:`toga.Screen` on which this window is present.
+
+        Returns:
+            The screen of :class:`toga.Screen` on which this window is present.
```

Comment:
We're moving away from Google format towards Sphinx autodoc format for docstrings; in addition, we don't need to declare a `returns` for properties.